packages/docs/components/page-configs/va-checkbox/api-options.ts

```diff
@@ -0,0 +1,40 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
Not really sure why checkbox had flown to the right:
![image](https://user-images.githubusercontent.com/5394573/82837558-1ae8b300-9ed2-11ea-93ef-ea6395f1490b.png)

Here's the example from demo:
![image](https://user-images.githubusercontent.com/5394573/82837545-0e645a80-9ed2-11ea-840f-11efa0d05fcd.png)

Reply:
Hm... I'll check